### PR TITLE
Mainsection/nextjs

### PR DESCRIPTION
**Pull request by VisionOra**
add main section 
make previous code responsive 
also add drop-down menu of top bar

### CODE REVIEW

**Comment by VisionOra** [comment]
image

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,27 @@
+node_modules/
+.next/
```
Comment (sonangrai) [inline .gitignore:2]: only .next/ will be enough to ignore inside files @SohaibAnwaar

File: components/common/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+const Header = () => {
+  return (
+    <HeaderSection>
+      <Head>
+      <script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>
+      <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.bundle.min.js"></script>
+      </Head>
+      <nav>
+      <div className="container-fluid bg-light position-relative shadow">
+  <nav className="navbar navbar-expand-lg bg-light navbar-light py-3 py-lg-0 px-0 px-lg-5">
```
Comment (sonangrai) [inline components/common/Header.tsx:13]: prettier not configured, configure prettier to prettify codes @SohaibAnwaar

File: components/common/Header.tsx [hunk 3]
```diff
@@ -0,0 +1,142 @@
+import Link from "next/link";
+import styled from "styled-components";
+import Head from 'next/head'
+const Header = () => {
+  return (
+    <HeaderSection>
+      <Head>
+      <script src="https://code.jquery.com/jquery-3.4.1.min.js"></script>
```
Comment (sonangrai) [inline components/common/Header.tsx:8]: Why jquery in React ? @SohaibAnwaar

File: components/common/Header.tsx [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+      <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.bundle.min.js"></script>
```
Comment (sonangrai) [inline components/common/Header.tsx:9]: we dont add library like this @SohaibAnwaar we add library from node modules or for cdn we add it on layout

File: components/common/Header.tsx [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+            <span className="dropdown-item-sub-module"><br/>Self-simple payment that easily interlocks</span>
+            </a>
+            <a href="#" className="dropdown-item">            
+            <span>Issuance of free cash receipts</span>
+            <span className="dropdown-item-sub-module"><br/>IRS issuance agency service</span>
+            </a>
+            </div>
+          </div>
+        </div>
+        <a href="usage_fee.html" className="nav-item nav-link">
```
Comment (sonangrai) [inline components/common/Header.tsx:85]: We have Link for a check on nextjs doc @SohaibAnwaar

File: components/common/Header.tsx [hunk 6]
```diff
@@ -0,0 +1,142 @@
+import Link from "next/link";
```
Comment (sonangrai) [inline components/common/Header.tsx:1]: this component is large we talked about atomic components @SohaibAnwaar

**Review by sonangrai — COMMENTED** [review]
@SohaibAnwaar remove unecessary files of cypress and check comments

**Pull request merged**